G3D10/G3D-app.lib/source/OptiXTriTree.cpp

```diff
@@ -456,9 +460,12 @@ void OptiXTriTree::intersectRays
 
         !(options & TriTree::NO_PARTIAL_COVERAGE_TEST),
         !(options & TriTree::DO_NOT_CULL_BACKFACES),
-		0, // materialLOD
-		mask
-    );
+        baseMipLevel, // materialLOD
+        notNull(rayCone) ? rayCone->glBufferID() : GL_NONE,
+        mask
+    )) {
+        alwaysAssertM(false, "Internal wave.lib exception printed to stderr. This is likely an out of memory exception due to copying too many textures into CUDA-acceptable formats.");
```

Comment:
This change in call signature seems to depend on a `wave.h` change that is not included with the current pull request. @ZanderMajercik my build currently fails because of this mismatch.

Reply:
I just noticed that I'm fully ignoring the entire `/G3D10/external` directory because it has a bunch of large files in it. Do we need those changes to be supported here? 

I was under the impression that most of the `external` directory would be bulk updates done in the main G3D, which is why I excluded it here, but it seems like `wave.lib` is considered external and has changes we need for this. I can un-ignore `wave.lib` but I believe it's rather large. Are the `wave.lib` changes minor enough that we could just commit them to the svn and depend on them for this pull request?